src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -80,13 +80,16 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public HttpClientHandler() { }
         public bool AllowAutoRedirect { get { return default(bool); } set { } }
         public System.Net.DecompressionMethods AutomaticDecompression { get { return default(System.Net.DecompressionMethods); } set { } }
+        public bool CheckCertificateRevocationList { get { return default(bool); } set { } }
         public System.Net.Http.ClientCertificateOption ClientCertificateOptions { get { return default(System.Net.Http.ClientCertificateOption); } set { } }
+        public System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates { get { return default(System.Security.Cryptography.X509Certificates.X509Certificate2Collection); } }
```

Comment:
>What's the rationale for having the API defined with a return type of X509CertificateCollection rather than X509Certificate2Collection? 

The problem is that we have an existing type in the .NET Framework, `WebRequestHandler`. This type lives in a separate DLL, System.Net.Http.WebRequest.dll.  And `WebRequestHandler` already has a property called `ClientCertificates` and it is on type `X509CertificateCollection`.

So, we can't really add a new property to `HttpClientHandler` with the same name of `ClientCertificates` but have it be of type `X509Certifcate2Collection`.  We are concerned about what happens if a developer take the OOB package of System.Net.Http.dll and also adds a reference to System.Net.Http.WebRequest.dll.  There would be conflicts in type of a similarly named property.